microcosm_postgres/models.py

```diff
@@ -80,6 +80,10 @@ def created_timestamp(self):
     def updated_timestamp(self):
         return (self.updated_at.replace(tzinfo=None) - EPOCH).total_seconds()
 
+    @classmethod
+    def sort_by(cls):
+        return cls.created_at.asc()
```

Comment:
I feel like we usually care more about more recent events, so maybe `dsc` is more appropriate?

Reply:
I used `asc` to ensure that while we paginate, new entities are always at the end so we don't run into race conditions when paginating.

Reply:
I checked the database schema of 40+ services and found that 95% of tables have **created_at** columns. Here is a list of tables missing the above column,
`- florentino_db.sql:CREATE TABLE public.external_write_permission (
 - florentino_db.sql:CREATE TABLE public.template_question_segment_association (
 - florentino_db.sql:CREATE TABLE public.template_segment_edge (
 - picasso_db.sql:CREATE TABLE public.shortlist_item (
 - prudencio_db.sql:CREATE TABLE public.client_legal_entity_item (
 - prudencio_db.sql:CREATE TABLE public.expiration_item (
 - prudencio_db.sql:CREATE TABLE public.fee_blended_rate_card_item (
 - prudencio_db.sql:CREATE TABLE public.fee_expenses_item (
 - prudencio_db.sql:CREATE TABLE public.fee_plan_item (
 - prudencio_db.sql:CREATE TABLE public.fee_platform_item (
 - prudencio_db.sql:CREATE TABLE public.fee_rate_card_item (
 - prudencio_db.sql:CREATE TABLE public.fee_taxes_item (
 - prudencio_db.sql:CREATE TABLE public.free_text_item (
 - prudencio_db.sql:CREATE TABLE public.payment_detail_item (
 - prudencio_db.sql:CREATE TABLE public.plan_duration_item (
 - prudencio_db.sql:CREATE TABLE public.plan_item (
 - prudencio_db.sql:CREATE TABLE public.privacy_policy_item (
 - prudencio_db.sql:CREATE TABLE public.procurement_category_item (
 - prudencio_db.sql:CREATE TABLE public.provider_legal_entity_item (
 - prudencio_db.sql:CREATE TABLE public.team_member_item (
 - prudencio_db.sql:CREATE TABLE public.terms_policy_item (
 - prudencio_db.sql:CREATE TABLE public.upload_item (`
 Table **alembic_version** is also missing the above column, since we use it for versioning purpose only I excluded it from above list.
2. I would recommend adding an index for **created_at** column for sort purpose, else it will be done at runtime.  Only some of the tables have the index on this column.
3. We need to ensure that new tables follow the convention of **created_at** column plus index.